components/shared/DropdownButton/dropdownButton.module.css

```diff
@@ -0,0 +1,37 @@
+.dropdownToggle {
+  border-radius: 8px;
+  font-family: "Nunito";
+  font-weight: 500;
+  line-height: 20px;
+  letter-spacing: 1.5%;
+  display: flex;
```

Comment:
```suggestion
  border-radius: 8px;
  display: flex;
```

these are inherited, you can omit them here

Reply:
Did you try checking this on your ide? Those styles aren't inherited and affect the css...

Reply:
if you are referring to the fact that font-family is inherited from the Faq component, I think that is fine because although in this case there might be a redundancy, the dropdown is a shared component so it needs to have its own font-family css
The other css stylings are needed (font-weight, etc) as the css styling that is inherited has different values for those css styles.